pkg/storage/fs/cephfs/cephfs.go

```diff
@@ -615,18 +624,208 @@ func (fs *cephfs) UpdateStorageSpace(ctx context.Context, req *provider.UpdateSt
 	return nil, errtypes.NotSupported("unimplemented")
 }
 
+var fnvHash hash.Hash32 = fnv.New32a()
```

Comment:
```suggestion
var fnvHash = fnv.New32a()
```

Type can be inferred in that declaration.